From: Andrew Deason <adeason@sinenomine.net>
Date: Wed, 17 Oct 2018 16:35:36 -0500
Subject: Remove one more automake VERSION reference

The configure summary was still referencing the old automake-specific
VERSION var. Use the autoconf PACKAGE_VERSION var instead, so this
actually shows our version.

Change-Id: I18007935d0235931f1d2e023abddee7356e8ac2d
Reviewed-on: https://gerrit.openafs.org/13360
Reviewed-by: Benjamin Kaduk <kaduk@mit.edu>
Tested-by: BuildBot <buildbot@rampaginggeek.com>
(cherry picked from commit 0f65b40b24599d58cf30bfd47fae83ab54e1416a)
---
 src/cf/summary.m4 | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/cf/summary.m4 b/src/cf/summary.m4
index 8e45413..2967b02 100644
--- a/src/cf/summary.m4
+++ b/src/cf/summary.m4
@@ -59,7 +59,7 @@ AC_DEFUN([OPENAFS_SUMMARY],[
 ***************************************************************
 OpenAFS configure summary
 
-  version : ${VERSION}
+  version : ${PACKAGE_VERSION}
   sysname : ${AFS_SYSNAME}
 
 debug:
